test/jobs/measure_evaluation_job_test.rb

```diff
@@ -8,33 +8,6 @@ def setup
                                       measure_ids: ['BE65090C-EB1F-11E7-8C3F-9A214CF093AE'])
   end
 
-  def test_can_use_sync_and_async_results
```

Comment:
Are we planning on restoring this or something like this at some point in time? (do we want a TODO here?)

Reply:
eh, I don't think so.  I don't really see the utility of this.